README.md

```diff
@@ -1,2 +1,35 @@
 # playground-for-php
```

Comment:
Consistency is key. Let's just replicate this README file: https://github.com/appwrite/playground-for-node replacing Node stuff with PHP.